### PR TITLE
Feature/issue 516 make button accomodate two lines of text

### PR DESCRIPTION
**Pull request by DarwishMenna**
@rasmus-storjohann-PG @bensinjin

### CODE REVIEW

File: src/components/tasks/task_detail_content_component.tsx [hunk 1]
```diff
@@ -130,6 +130,14 @@ const ServicesButton = (props: Props): JSX.Element => (
...
         }}
     >
+    <View style={{
+        flex: 1,
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        alignSelf: 'center',
+        marginHorizontal: 50,
+    }}>
```
Comment (bensinjin) [inline src/components/tasks/task_detail_content_component.tsx]: This has too much margin now. Android is apparently buggy when it comes to padding. What I think we should do here is something like: 

```
     import { Image, Dimensions, Platform } from 'react-native';
     ...
    <MultiLineButtonComponent
        onPress={props.onServicesTextPress}
        additionalStyles={{
            marginBottom: 15,
            paddingHorizontal: Platform.OS === 'ios' ? 15 :  0,
        }}
    >
    <View style={{
        flex: 1,
        flexDirection: 'row',
        justifyContent: 'center',
        alignItems: 'center',
        alignSelf: 'center',
        marginHorizontal: Platform.OS === 'ios' ? 0 :  20,
    }}>
       ...
    </View>
```

This looks a little better. I'm not sure we'll get perfect on this but this is what I see after these changes:

French iOS left, Android right:
Screen Shot 2019-03-18 at 1 07 38 PM

English iOS left, Android right:
Screen Shot 2019-03-18 at 1 08 53 PM

**Comment by bensinjin** [comment]
@DarwishMenna is there a reason you're using a fork of this repo?

File: src/components/tasks/task_detail_content_component.tsx (hunk 1 above)
Comment (DarwishMenna) [reply src/components/tasks/task_detail_content_component.tsx]: I will try that. Thanks Ben

**Comment by rasmus-storjohann-PG** [comment]
@bensinjin about the forking, it was my call when @DarwishMenna just started, I wanted to try out the fork repo work flow vs. working in the same repo. I wanted to try that out in anticipation of getting volunteers working in there. Which didn't happen. We'll fix this.

**Comment by bensinjin** [comment]
@DarwishMenna this is looking good, last thing, please re add the `paddingHorizontal: 15` to `multiline_button_component.tsx` So this looks normal in other places.

**Comment by bensinjin** [comment]
@rasmus-storjohann-PG this is ready to go - feel free to merge it when you're ready.

**Pull request merged**